src/workerd/jsg/commonjs.h

```diff
@@ -138,8 +132,6 @@ class NodeJsModuleContext: public jsg::Object {
     JSG_METHOD(require);
     JSG_READONLY_INSTANCE_PROPERTY(module, getModule);
     JSG_INSTANCE_PROPERTY(exports, getExports, setExports);
-    JSG_LAZY_INSTANCE_PROPERTY(Buffer, getBuffer);
-    JSG_LAZY_INSTANCE_PROPERTY(process, getProcess);
```

Comment:
Why did we remove these?

Reply:
See the first bullet point in the PR description.